database/database.go

```diff
@@ -184,6 +194,25 @@ func FetchRecords(tableName string) (map[string]string, error) {
 	return getCurrentDB()[FETCH_ALL].(func(string) (map[string]string, error))(tableName)
 }
 
+// InitializeUUID - create a UUID record for server if none exists
+func InitializeUUID() error {
```

Comment:
should be private and add to InitializeDatabase func

Reply:
made private and added to initialize